src/markup.rs

```diff
@@ -0,0 +1,88 @@
+mod html;
+mod markdownv2;
+
+use reqwest::Url;
+
+use crate::types::{User, UserId};
+
+pub use html::Html;
+pub use markdownv2::MarkdownV2;
+
+/// Allows formatting text according to one of the markup languages
+/// supported by Telegram.
+pub trait Markup {
```

Comment:
@Hirrolot what do you think of this?

Having a trait allows us to unify the methods between html & markdown.
This also allows users to be abstract over markup language, though I'm unsure how useful this is.

The downside is that all methods take `&self`, so you need to write `MarkdownV2.bold(...)` instead of `use ...::markdownv2::bold;` and then `bold(...)`. Is this too bad, should we add top level functions back? wdyt?

Reply:
I don't quite get the purpose of this trait. Yes, theoretically, it allows you to abstract over a markup, but honestly, I don't think it's needed in real-world Telegram bots.

What's wrong with freestanding functions as in teloxide right now?

Reply:
Nothing wrong with freestanding functions really. I mostly added this trait to make sure that the HTML and MarkdownV2 interfaces are kept in sync and to implement this for `ParseMode`.

Reply:
And well, this also solves the problem of importing function for the wrong markup, though I'm not sure how big/real this problem is.

Reply:
Ok, let's just leave the freestanding functions as-is. Plus we wouldn't have the problem with `// "best effort"` for Markdown V1.